src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -82,6 +82,13 @@ public UniquePersonList(UniquePersonList source) {
     }
 
 
+    /**
+     * Returns the count of the list.
+     */
+    public int count() {
```

Comment:
Maybe better to follow the style of naming for the other stuff and call this `getCount`?